src/pages/settings-general/settings-general.ts

```diff
@@ -50,6 +54,19 @@ export class SettingsGeneralPage {
     this.settingsService.saveNightModeSettings(this.selectedTheme);
   }
 
+  /**
+   * Called when pressing the funds switch
+   */
+  fundsSwitch(): void {
+    if (this.showFundsStatus) {
+      this.settingsService.setFundsSwitchStatus(true);
```

Comment:
Couldn't saving the settings be implicit when setting the fund switch status? Is there a case in which you want to set the fund switch status and not save the result immediately?